src/Hooks/useAnime.js

```diff
@@ -21,7 +20,7 @@ export default function useAnime(animeId, cachedAnime) {
     // Otherwise, fetch anime from API.
     setFetchedAnime(undefined);
     setError(undefined);
-    APIGetAnime(animeId)
+    APIGetAnime(animeId, ignoreCall)
```

Comment:
it would be cool if we could do this without calling the APIGetAnime function.  maybe something up above near the other early-out.  consider something like: 
```js
    // Don't load if we have the right data already.
    if (getMatchingAnime(animeId, [cachedAnime, fetchedAnime])) {
      return;
    }
    // Don't send API request if told to ignore it.
    if (ignoreCall) {
      return;
    }
```